lib/config/screen.ex

```diff
@@ -20,6 +20,7 @@ defmodule ScreensConfig.Screen do
           | :solari_large
           | :solari_large_v2
           | :pre_fare_v2
+          | :elevator_v2
 
   @type t :: %__MODULE__{
           vendor: :gds | :mercury | :solari | :c3ms | :outfront | :lg_mri,
```

Comment:
We should add the new vendor for these screens: `:mimo` (and make sure this is selectable as a vendor when adding elevator screens in the `screens` admin UI).

Reply:
I see your latest commit says "add new vendor" but this line remains unchanged. Forgot to commit something? 🙂

Reply:
Oops I did forget to commit 😅 thank you for catching that!

Reply:
I also added the vendor to [line 125](https://github.com/mbta/screens-config-lib/pull/36/files#diff-126737edc2fde54ebb834201e7b8c48160e112edb8fc6428ee7be6cee6b97184L122) as assume that's a necessary change too?